vms/rpcchainvm/vm_server.go

```diff
@@ -603,26 +591,17 @@ func (vm *VMServer) AppRequest(ctx context.Context, req *vmpb.AppRequestMsg) (*e
 }
 
 func (vm *VMServer) AppRequestFailed(ctx context.Context, req *vmpb.AppRequestFailedMsg) (*emptypb.Empty, error) {
-	nodeID, err := ids.ToNodeID(req.NodeId)
-	if err != nil {
-		return nil, err
-	}
+	nodeID := ids.NodeIDFromBytes(req.NodeId, len(req.NodeId))
```

Comment:
This should error if the bytes aren't 32 or 20

Reply:
Done